lib/contentful_converter.rb

```diff
@@ -3,7 +3,12 @@
 require 'contentful_converter/converter'
 
 module ContentfulConverter
-  def self.convert(html)
+  def self.convert(html, modifiers = {})
+    @@modifiers = modifiers
```

Comment:
It could be worth exploring other options to avoid using class variables e.g. instance variables, constants etc?, as class variables can end up causing unexpected behaviour if accidentally reassigned further down the line